st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -192,6 +180,30 @@ def run(self, action_parameters):
             }
         }
 
+        return options
+
+    def _get_resume_options(self):
+        return self.context.get('re-run', {})
+
+    @retrying.retry(
+        retry_on_exception=utils.retry_on_exceptions,
+        wait_exponential_multiplier=cfg.CONF.mistral.retry_exp_msec,
+        wait_exponential_max=cfg.CONF.mistral.retry_exp_max_msec,
+        stop_max_delay=cfg.CONF.mistral.retry_stop_max_msec)
+    def run(self, action_parameters):
+        resume_options = self._get_resume_options()
+        tasks = resume_options.get('tasks', [])
+        resume = self.rerun_ex_ref and tasks
+        return self.resume(self.rerun_ex_ref, tasks) if resume else self.start(action_parameters)
```

Comment:
This is just a minor style thing - short hand if notation usually only tends to be used for simple variable assignments.

For cases like this it's usually more readable if you have an explicit if and else.